# Install python extension in the install directory of OpenMM-PLUMED
# wpoely86@gmail.com
diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index 5e76b5e..f449e75 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -22,6 +22,6 @@ set(OPENMMPLUMED_LIBRARY_DIR "${CMAKE_BINARY_DIR}")
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/setup.py ${CMAKE_CURRENT_BINARY_DIR}/setup.py)
 add_custom_command(TARGET PythonInstall
     COMMAND "${PYTHON_EXECUTABLE}" setup.py build
-    COMMAND "${PYTHON_EXECUTABLE}" setup.py install
+    COMMAND "${PYTHON_EXECUTABLE}" setup.py install --prefix ${CMAKE_INSTALL_PREFIX}
     WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
 )
